diff --git a/node_modules/antlr4ts/Recognizer.js b/node_modules/antlr4ts/Recognizer.js
index 8c55216..289a33b 100644
--- a/node_modules/antlr4ts/Recognizer.js
+++ b/node_modules/antlr4ts/Recognizer.js
@@ -21,7 +21,7 @@ const Token_1 = require("./Token");
 const Utils = require("./misc/Utils");
 class Recognizer {
     constructor() {
-        this._listeners = [ConsoleErrorListener_1.ConsoleErrorListener.INSTANCE];
+        this._listeners = [];
         this._stateNumber = -1;
     }
     /**
