--- a/Portugol.g4
+++ b/Portugol.g4
@@ -149,8 +149,7 @@
     |   expressao OP_MAIOR_IGUAL expressao                                                      #operacaoMaiorIgual
     |   expressao OP_E_LOGICO expressao                                                         #operacaoELogico
     |   expressao OP_OU_LOGICO expressao                                                        #operacaoOuLogico
-    |   expressao OP_SHIFT_LEFT expressao                                                       #operacaoShiftLeft
-    |   expressao OP_SHIFT_RIGHT expressao                                                      #operacaoShiftRight
+    |   expressao op=(OP_SHIFT_LEFT | OP_SHIFT_RIGHT) expressao                                 #operacaoShift
     |   expressao E_COMERCIAL expressao                                                         #operacaoAndBitwise
     |   expressao OP_XOR expressao                                                              #operacaoXor
     |   expressao OP_OU_BITWISE expressao                                                       #operacaoOrBitwise
